modules/test/playwright/tests/object-web/objectRelationships.spec.ts

```diff
@@ -42,16 +42,19 @@ test.describe('Manage object relationships through Model Builder', () => {
 
 		await viewObjectDefinitionsPage.viewInModelBuilder();
 
-		await modelBuilderPage.clickToggleSidebarsButton();
+		await modelBuilderPage.toggleSidebarsButton.click();
```

Comment:
I noticed that the clickToggleSidebarsButton method was asynchronous but did not have the wait flag.

That was the main reason for the flaky test, right? Good analysis, well done :)

Reply:
As we discussed, I actually didn't think this could be the issue since We have other methods working like this throughout our pages, but to avoid future possible flakiness, I've added the wait flag where it was missing and simplified the click methods for ViewObjectDefinitions and ModelBuilder pages. Could you please review it again? thanks for your insight! @igor-franca